cola/algorithms/diagonal_estimation.py

```diff
@@ -98,7 +98,7 @@ def exact_diag(A: LinearOperator, k=0, bs=100, tol=3e-2, max_iters=10000, pbar=F
 
 
 @export
-def approx_diag(A: LinearOperator, k=0, bs=100, tol=3e-2, max_iters=10000, pbar=False):
+def approx_diag(A: LinearOperator, k=0, bs=100, tol=3e-2, max_iters=10000, hutchpp_dist="rademacher", pbar=False):
```

Comment:
Maybe lets separate this kwarg into
`hutchpp=False`
`probe_dist='normal'` or `'rademacher'`
to keep things more straightforward